### DISCUSSION TITLE
“保交楼”复产复工30天！你们的房子有消息没？

### DISCUSSION THREAD

**Discussion by caotianchao** [WeNeedHome/SummaryOfLoanSuspension#1184] [General]
据郑州发布消息，10月6日，距郑州市印发《“大干30天，确保全市停工楼盘全面复工”保交楼专项行动实施方案》整30天的时间节点，郑州全市已排查出的147个已售停工、半停工商品住宅项目中，有145个实现全面、实质性复工，郑州“保交楼、稳民生、促发展”专项行动取得重要阶段性成果。

相关人员表示，2个未复工的项目目前已确定化解路径：新密市凯旋山项目进入破产重整程序，新郑市盛华学府项目开始破产清算，预计十一月底前可实现复工。

**Comment by Swwwwift** [comment]
插眼。刚刚刷到一个视频说招聘人，只要会敲钢管，表演复工。一天100

**Reply by Swwwwift** [reply]
不知真假
